### PR TITLE
CSS color tweaks

### PR DESCRIPTION
**Pull request by silverwind**
Change grey shades in arc-green to match the theme more:

Screenshot 2023-03-30 at 21 42 34

Adjusted grey shade in light theme:

image

Increase contrast in arc-green, change background to slightly darker shade, change forgeground to slightly brighter colors:

Screenshot 2023-03-30 at 22 33 20

Increase contrast of grey text in light theme as well by making them darker:

Screenshot 2023-03-30 at 22 33 35

Add color rule for border multiple select items:

Screenshot 2023-03-30 at 22 29 31
Screenshot 2023-03-30 at 22 29 46

Added color rule for red `*` on required form fields:

image

### CODE REVIEW

File: web_src/css/base.css [hunk 1]
```diff
@@ -92,9 +91,10 @@
   --color-purple-light: #bb64d8;
   --color-pink-light: #e86bb1;
   --color-brown-light: #c58b66;
-  --color-grey-light: #a6a6a6;
   --color-black-light: #525558;
   /* other colors */
+  --color-grey: #5a5a5a;
+  --color-grey-light: #838383;
```
Comment (silverwind) [inline web_src/css/base.css:97]: These were moved because in arc-green, they don't obey the comment above.

**Review by delvh — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Comment by silverwind** [comment]
Slight issue, the grey change in light theme makes these number labels too dark:

image

I'll adjust them. Ultimately, I think we need a separate color for them as these grey shades are not used consistently in the UI, sometimes for text, sometimes for background.

**Comment by silverwind** [comment]
Lightened a bit, wouldn't go much further as they become low-contrast otherwise:

image

**Comment by delvh** [comment]
No, I'm also rather a fan of having a fairly high contrast there as it was previously.
I preferred the primary color previously the most (probably to some part because I'm accustomed to them), but your proposal is the second best option.

**Comment by silverwind** [comment]
I now like these new grey tones more than previous primary. There is one inconsistency in PR tabs that still use primary, we need to eliminate that later 😉

File: web_src/css/themes/theme-arc-green.css [hunk 2]
```diff
@@ -124,19 +124,19 @@
...
+  --color-text-light-1: #aaafb9;
+  --color-text-light-2: #9a9ea9;
+  --color-text-light-3: #8a8e99;
   --color-footer: #2e323e;
   --color-timeline: #4c525e;
-  --color-input-text: #d5dbe6;
+  --color-input-text: #dfe3ec;
   --color-input-background: #232933;
   --color-input-toggle-background: #454a57;
   --color-input-border: #454a57;
```
Comment (wxiaoguang) [inline web_src/css/themes/theme-arc-green.css:142]: I guess the input-border color is too dark, I can not find the radioboxes. 🤣 

image

Comment (silverwind) [reply web_src/css/themes/theme-arc-green.css:142]: Yeah that needs improvement or convertion to native :)

**Pull request merged**